include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -75,7 +76,8 @@ class CMAES
         const size_t batchSize = 32,
         const size_t maxIterations = 1000,
         const double tolerance = 1e-5,
-        const SelectionPolicyType& selectionPolicy = SelectionPolicyType());
+        const SelectionPolicyType& selectionPolicy = SelectionPolicyType(),
+        const double initialSigma = 0.6);
```

Comment:
Hm, do you think we could just use `sigma` instead of `initialSigma` here?

Reply:
I used this parameter to tackle point number 2 in #70. If I only used sigma as you suggested here then it would have the same name as [this](https://github.com/mlpack/ensmallen/blob/master/include/ensmallen_bits/cmaes/cmaes_impl.hpp#L80) `arma` vector here and will cause an error. Please correct me if I am wrong somewhere.